soroban-built-in-token-spec/src/lib.rs

```diff
@@ -0,0 +1,152 @@
+#![no_std]
+use soroban_sdk::{contractimpl, BigInt, Bytes, Env};
+
+use soroban_auth::{Identifier, Signature};
+
+mod public_types;
+use crate::public_types::{ClassicMetadata, TokenMetadata};
+
+pub trait TokenTrait {
```

Comment:
Fyi the trait isn't required, you can just have the contract be an zero type with the impl. The trait has no impact on the contract spec.

Reply:
Good to know. Removed the trait.